test/app_automate_driver_test.rb

```diff
@@ -12,7 +12,27 @@ class AppAutomateDriverTest < Test::Unit::TestCase
   TARGET_DEVICE = "ANDROID_9"
   LOCAL_TUNNEL_COMMAND = "/BrowserStackLocal -d start --key #{ACCESS_KEY} --local-identifier #{LOCAL_ID} --force-local"
 
+  def setup
```

Comment:
Do we really need unit tests for the maze runner code at all?  It feels like we're testing the tests and that time spent maintaining these tests is nugatory.  If we shipped Maze Runner as a product it might be different, but it's only used for Bugsnag CI purposes and to my mind the test harness is verified by virtue of the number of CI tests being run with it.

Reply:
This is true to an extent, however especially when interacting with BrowserStack through Appium it's useful to be able to verify that everything is working as expected in the majority of circumstances.
At the moment our integration tests explicitly testing Maze-runner in use are fairly light, but we could move to focus on them rather than unit tests.

Reply:
It's also good to rule out any unexpected behaviour that may occur only rarely in maze-runner use.

Reply:
Ok, if we feel we're getting value out of them then that seems perfectly sensible and I don't have a problem with targeted tests.  I just wouldn't want to slavishly write unit tests for everything "because we do".

Reply:
Agreed, I don't think unit testing all logic is necessary in most cases.  We probably only need "complete" unit testing around tricky bits of functionality which connects to external services, with light testing around the rest.